src/lib/message.ts

```diff
@@ -56,6 +60,7 @@ class GleeMessage extends EventEmitter {
     if (serverName) this._serverName = serverName
     if (connection) this._connection = connection
     if (broadcast) this._broadcast = !!broadcast
+    if (cluster) this._cluster = !!cluster
```

Comment:
I think it's safe to do this:

```suggestion
    if (cluster) this._cluster = cluster
```

Since TypeScript is taking care that this is always a boolean.